src/pages/office/office.actions.ts

```diff
@@ -1,10 +1,17 @@
 import { Action } from '@ngrx/store';
 
 export const OFFICE_VIEW_DID_ENTER = '[OfficePage] Office view did enter';
+export const OFFICE_VIEW_DID_ADD_DANGEROUS_FAULT_COMMENT = '[OfficePage] Office view did add dangerous fault comment';
```

Comment:
`DID` in the action above (perhaps incorrectly) refers to a lifecycle hook. I'd leave that out here and just go with `ADDED_DANGEROUS_FAULT_COMMENT` (but leaving the page prefix on the value).